packages/kit/src/exports/vite/dev/index.js

```diff
@@ -20,6 +20,8 @@ import { SCHEME } from '../../../utils/url.js';
 import { check_feature } from '../../../utils/features.js';
 
 const cwd = process.cwd();
+// vite-specifc queries that we should skip handling for css urls
+const vite_css_query_regex = /(?:\?|&)(?:raw|url|inline)(?:&|$)/;
```

Comment:
can we use propper parsing here? a little worried raw= or inline= would get missed by this. maybe sufficient to include = in the last group?

Reply:
can we import it then at least? just the other day we had an issue in vps due to regex from different package going out of sync.

but ultimateley there are 2 apis available with querystring and urlsearchparams, so why regex?

Reply:
Vite had used this regex for years, it's unlikely to change again. And why not regex? There's different ways to do the same thing and regex just seemed faster and consistent with vite. (Respectfully but) I don't think it's worth nitpicking this.

Reply:
not dying on this hill but these things add up over time.